### PR TITLE
Add Cookstyle 6.3 notes

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
+## Cookstyle 6.3
+
+### 2 New Cops
+
+#### ChefModernize/RespondToCompileTime
+
+The `ChefModernize/RespondToCompileTime` detects gating usage of the `compile_time` property in the `chef_gem` resource with `if respond_to?(:compile_time)`. The compile_time property was added in Chef Infra Client 12.1 and can safely be used without checking first.
```
Comment (IanMadd) [inline RELEASE_NOTES.md:7]:
```suggestion
The `ChefModernize/RespondToCompileTime` cop detects gating usage of the `compile_time` property in the `chef_gem` resource with `if respond_to?(:compile_time)`. The `compile_time` property was added in Chef Infra Client 12.1 and can safely be used without checking first.
```

Comment (phiggins) [inline RELEASE_NOTES.md:7]:
```suggestion
The `ChefModernize/RespondToCompileTime` cop detects gating usage of the `compile_time` property in the `chef_gem` resource with `if respond_to?(:compile_time)`. The compile_time property was added in Chef Infra Client 12.1 and can safely be used without checking first.
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+#### ChefDeprecations/DeprecatedShelloutMethods
+
+The `ChefDeprecations/DeprecatedShelloutMethods` cop detects the deprecated shell_out helper methods that were removed in Chef Infra Client 15. These helpers were not widely used and were never publically documented, but they are still occasionally found in cookbooks.
```
Comment (phiggins) [inline RELEASE_NOTES.md:15]:
```suggestion
The `ChefDeprecations/DeprecatedShelloutMethods` cop detects deprecated `shell_out` helper methods that were removed in Chef Infra Client 15. These helpers were not widely used and were never publicly documented, but they are still occasionally found in cookbooks.
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
+### RuboCop 0.82
+
+The RuboCop engine that powers Cookstyle has been updated from 0.81.0 to 0.82.0 which includes several important bug fixes that impacted cookbooks.
```
Comment (IanMadd) [inline RELEASE_NOTES.md:23]:
```suggestion
The RuboCop engine that powers Cookstyle has been updated from 0.81.0 to 0.82.0. This update includes several important bug fixes that impacted cookbooks.
```

File: RELEASE_NOTES.md (hunk 1 above)
Comment (tas50) [reply RELEASE_NOTES.md]: Fixed

**Pull request merged**